VideoLocker/src/main/java/org/edx/mobile/view/CourseDashboardFragment.java

```diff
@@ -31,12 +36,90 @@ public void onCreate(Bundle savedInstanceState) {
 
     }
 
-    @SuppressLint("SetJavaScriptEnabled")
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
             Bundle savedInstanceState) {
         View view = inflater.inflate(R.layout.fragment_course_dashboard, container,
                 false);
+        courseTextName = (TextView) view.findViewById(R.id.course_detail_name);
+        courseTextDetails = (TextView) view.findViewById(R.id.course_detail_extras);
+
+        //Implementation Note: - we can create a list view and populate the list.
+        //but as number of rows are fixed and each row is different. the only common
+        //thing is UI layout. so we reuse the same UI layout programmatically here.
+        LinearLayout parent = (LinearLayout)  view.findViewById(R.id.dashboard_detail);
+
+        View custom = inflater.inflate(R.layout.row_course_dashboard_list, null);
```

Comment:
@aleffert  changed.  to use viewholder to make less duplicated code.  the problem with list of ids is that we also need to create two lists of Strings.